pkg/frontend/adminactions/resources_list.go

```diff
@@ -159,6 +159,24 @@ func (a *azureActions) appendAzureNetworkResources(ctx context.Context, armResou
 					Resource: rt.RouteTable,
 				})
 			}
+			//Due to BYO NSGs not belonging to the managed resource group, we would like to list them as well
+			if snet.Properties.NetworkSecurityGroup.ID != nil {
+				r, err := azure.ParseResourceID(*snet.Properties.NetworkSecurityGroup.ID)
```

Comment:
I'd consider renaming `r` to something more descriptive like `networkSecurityGroup` or just `nsg`.